bindings/rust/standard/s2n-tls-hyper/Cargo.toml

```diff
@@ -14,8 +14,8 @@ default = []
 [dependencies]
 s2n-tls = { version = "=0.3.9", path = "../../extended/s2n-tls" }
 s2n-tls-tokio = { version = "=0.3.9", path = "../../extended/s2n-tls-tokio" }
-hyper = { version = "1" }
-hyper-util = { version = "0.1", features = ["client-legacy", "tokio", "http1", "http2"] }
+hyper = { version = "1.4" }
+hyper-util = { version = "0.1.4", features = ["client-legacy", "tokio", "http1", "http2"] }
```

Comment:
hyper-util 0.1.4 is needed because [we rely on Connection being implemented for TokioIo](https://github.com/aws/s2n-tls/blob/1c38961ddca45243b73c684dd4ddcc9ae454b2e3/bindings/rust/standard/s2n-tls-hyper/src/stream.rs#L50), which was [implemented in 0.1.4](https://github.com/hyperium/hyper-util/commit/7bae87f0fd1109e3ef48b449f63d045d67efba73).